languages/en_US.po

```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
 msgid "下一篇"
 msgstr "Next"
 
+#: admin_style.php:128
+msgid "文章页"
+msgstr "Posts"
+
+#: admin_style.php:129
+msgid "文章列表"
+msgstr "Post List"
+
+#: admin_style.php:130
+msgid "标题栏"
+msgstr "Blog Title"
+
+#: admin_style.php:389
+msgid "自动隐藏标题栏"
+msgstr "Automatically hide the title"
+
+#: admin_style.php:395
+msgid "开启后，页面向下滚动时标题栏会向上隐藏，向上滚动时会出现。"
+msgstr ""
+"The title bar will automatically hide when scrolling down, "
+"it will reappear on scrolling up."
+
+#: admin_style.php:400
+msgid "标题栏显示内容"
+msgstr "Title bar displays"
+
+#: admin_style.php:426
+msgid "点击标题栏返回顶部"
+msgstr "Return to top on click"
+
+#: admin_style.php:432
+msgid "开启后，点击标题栏可以返回页面顶部。此设置影响所有页面。</strong>"
+msgstr "Click the title bar after opening to return to top."
+
+#: admin_style.php:502
+msgid "如使用自定义 API，你需要在括号内填写 API URL 并确保 API 返回 <code>{\"text\":\"句子\"}</code> 的 JSON 格式。<a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">详细信息</a></code>"
+msgstr "If you are using a custom API, you'll need to fill-in the API URL in parentheses and make sure that the API returns the JSON format <code>{\"text\":\"quote\"}</code> <a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">details</a></code>"
+
+#: admin_style.php:500
+msgid "使用一言 API（自定义）"
+msgstr "Use a quote API (custom)"
+
+#: admin_style.php:132
+msgid "页脚"
+msgstr "Footer"
+
+#: admin_style.php:133
+msgid "Touch Bar 图标"
+msgstr "Touch Bar Icon"
+
+#: admin_style.php:134
+msgid "杂项"
+msgstr "Misc"
+
+#: admin_fn.php:92
+msgid "文章"
+msgstr "Article"
+
+#: 
+msgid "首页推荐文章获取方式"
+msgstr "get recommended articles from"
+
+#: 
+msgid "某一分类"
+msgstr "A certain category"
+
+#: 
+msgid "置顶文章"
+msgstr "Top articles"
+
+#: 
+msgid "在此设定首页推荐文章的获取方式。<br>若选择置顶文章，当没有置顶文章时，首页推荐文章模块将不会显示，同时文章列表将保持原始顺序而不会被置顶文章打乱。"
+msgstr ""
+"Set the source of recommended articles. If you select \"Top articles\" when there is no top article, then the recommended "
+"article module will not be displayed and the article list will remain in the original order."
+
+#: 
+msgid "在此设定首页推荐文章的分类名。当分类不存在时，将显示最新文章。"
+msgstr ""
+"Set the name of the category where the recommended articles should be fetched from. "
+"The latest articles will be displayed when the category does not exist."
+
+#: 
+msgid "评论"
+msgstr "Comments"
+
+#: 
+msgid "“发送评论”文本自定义"
+msgstr "<q>Send Comment</q> text customization"
+
+#: 
+msgid "在此自定义“发送评论”按钮上的文本。"
+msgstr "Customize the text on the <q>Send Comment</q> button."
+
+#: 
+msgid "无障碍"
+msgstr "Accessibility"
+
+#: 
+msgid "广告"
+msgstr "Advertising"
```

Comment:
Personally, I'd prefer to use `advertisements`, or `ads` in short. 
As `advertising` are more commonly used when we are selling our own products, while in this case we are including ads from other people selling their products on our website.
Correct me if I'm wrong, English is not my native language : )